ui/main/src/app/business/view/timeline/circles.ts

```diff
@@ -0,0 +1,246 @@
+/* Copyright (c) 2018-2023, RTE (http://www.rte-france.com)
```

Comment:
Copyright for new files should be just for 2023

Reply:
keep it as it contains code from a previous existing file